amethyst_utils/src/ortho_camera.rs

```diff
@@ -5,39 +5,96 @@ use amethyst_core::specs::{
 use amethyst_core::Axis2;
 use amethyst_renderer::{Camera, ScreenDimensions};
 
+/// The coordinates that `CameraOrtho` will keep visible in the window
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
Could you add `Copy` and `PartialEq` as well?
That'll make it more ergonomic to use, and easier to assert in tests